Kernel/Bus/PCI/Controller/HostBridge.cpp

```diff
@@ -53,7 +53,7 @@ UNMAP_AFTER_INIT Vector<Capability> HostBridge::get_capabilities_for_function(Bu
 UNMAP_AFTER_INIT void HostBridge::enumerate_functions(Function<void(DeviceIdentifier)> const& callback, BusNumber bus, DeviceNumber device, FunctionNumber function, bool recursive_search_into_bridges)
 {
     dbgln_if(PCI_DEBUG, "PCI: Enumerating function, bus={}, device={}, function={}", bus, device, function);
-    Address address(0, bus.value(), device.value(), function.value());
```

Comment:
The title of this commit seems a bit confusing, Do you mean "Don't hardcode address domain to 0 when enumerating devices"?